docs/tutorials/customizing-the-output/script-inject.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: default
-title: "Custom transformer for injecting a script into the container image"
-permalink: /tutorials/migration-workflow/script-inject
-parent: "Migration workflow"
+title: "Injecting a script into the container image"
```

Comment:
Do we want to call it as something like, adding custom files to source

Reply:
done